config/initializers/training_routes.rb

```diff
@@ -16,4 +19,19 @@
   TRAINING_ROUTE_ENUMS[:school_direct_tuition_fee] => 3,
   TRAINING_ROUTE_ENUMS[:school_direct_salaried] => 4,
   TRAINING_ROUTE_ENUMS[:pg_teaching_apprenticeship] => 5,
-}.with_indifferent_access.freeze
+  TRAINING_ROUTE_ENUMS[:early_years_assessment_only] => 6,
+  TRAINING_ROUTE_ENUMS[:early_years_graduate_entry] => 7,
+  TRAINING_ROUTE_ENUMS[:early_years_graduate_placement] => 8,
+}.freeze
+
+TRAINING_ROUTES_FOR_TRAINEE = TRAINING_ROUTES.select { |training_route|
```

Comment:
Doesn't this mean that in order to turn on a route e.g. `school_direct_salaried`, I need to set the feature to `true` as before but _also_ add it to this list?

If I forget to add it to `TRAINING_ROUTES_FOR_TRAINEE`, the feature might be turned on, but we're still not allowing `school_direct_salaried` trainees to be saved?

Reply:
We haven't developed those routes yet...

We only really have the choice of hard decisions such as turning it all `on`  or as we iterate towards `TRAINING_ROUTES` and selectively turn it `on`.

Reply:
Understood!